rod_cutting_problem/rod_cutting.c

```diff
@@ -0,0 +1,31 @@
+/*
+*  Given a rod of length n length unit and an array of prices that contains prices of all pieces of size smaller than n. 
+*  Determine the maximum new_pricesue obtainable by cutting up the rod and selling the pieces.
+*  Time Complexity: O(n^2)
+*  Space Complexity: O(n)
+*/
+
+#include <stdio.h>
+#include <string.h>
+
+#define max(a, b) (a > b)? a : b;
+
+int rod_cutting(int *prices) {
+    int i, j;
+    int n = sizeof(prices);
```

Comment:
It should be unsigned. Else you'll be passing `int` as 3rd argument instead of `unsigned int` in line 17.